### DISCUSSION TITLE
有Safari 用户的吗

### DISCUSSION THREAD

**Discussion by FoundTheWOUT** [the1812/Bilibili-Evolved#1835] [提问 / Q&A]
想知道这个插件能不能知道用户浏览器比例（有没有safari 用户🤔

**Comment by the1812** [comment]
感觉比较少 (根据以往的 issues) ~~毕竟 mac 独占~~
具体数据是多少我也不知道, 毕竟不收集任何用户数据

**Reply by FoundTheWOUT** [reply]
主要是最近用safari 遇到一些无伤大雅的问题，用的人不多且没人反馈就先随它吧，我慢慢收集一下问题，~~有空就修复一下~~。
